pkg/cmd/util/output/backup_describer.go

```diff
@@ -97,6 +97,9 @@ func DescribeBackupSpec(d *Describer, spec v1.BackupSpec) {
 	}
 	d.Printf("Label selector:\t%s\n", s)
 
+	d.Println()
+	d.Printf("Storage Location:\t%s\n", spec.StorageLocation)
```

Comment:
This is the primary change for adding storage location. The rest of the file should be the `v1` -> `arkv1api` change, unless I made a mistake.